firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -586,7 +586,12 @@ class Tab: NSObject, ThemeApplicable, FeatureFlaggable, ShareTab {
 
     func restore(_ webView: WKWebView, interactionState: Data? = nil) {
         if let url = url {
-            webView.load(URLRequest(url: url))
+            if let internalURL = InternalURL(url),
+               internalURL.isAboutHomeURL {
+                webView.load(PrivilegedRequest(url: url) as URLRequest)
+            } else {
+                webView.load(URLRequest(url: url))
+            }
```

Comment:
When I tested launching the app with the recurring blank screen, this code was enough to eliminate the issue! It automatically appended the UUID to make it an authorized internal home URL.

Reply:
Good catch @lmarceau !! 🎉

Reply:
Thanks @ih-codes & @lmarceau for the deep investigation on this 🚀